CONTRIBUTING.rst

```diff
@@ -67,3 +67,39 @@ Before committing please run the following code in the main directory:
 
 The style is checked with `flake8`. Also follow the commit message recommendations, 
 cf. `Writing good commit messages <https://github.com/erlang/otp/wiki/writing-good-commit-messages>`_.
+
+Examples
+-------
+
+Adding new SPARQL queries to Scholia
+
+1. Assign the task for yourself (if it is in an issue tracker)
+2. Get the new query e.g. 
+E.g. 
+SELECT DISTINCT ?author ?authorLabel ?award ?awardLabel WHERE {
```

Comment:
There is a bit of a formatting issue with the code. Perhaps some indentation is needed?

Reply:
looks not as intended... ill check!